packages/unlock-js/src/web3Service.ts

```diff
@@ -89,7 +89,7 @@ export default class Web3Service extends UnlockService {
       address,
       this.providerForNetwork(network)
     )
-    const lock = version.getLock.bind(this)(
+    const lock = await version.getLock.bind(this)(
```

Comment:
@julien51 looks like `await` is missing here, thats why `address` is not present.

Reply:
Very good catch